### PR TITLE
Feature/move to absolute imports

### PR DESCRIPTION
**Pull request by osherdp**

### CODE REVIEW

**Comment by osherdp** [comment]
Relates to #33 - not a complete fix, but should at least give full traceback in certain cases

File: src/rotest/core/utils/common.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from rotest.core.block import TestBlock
 from rotest.core.suite import TestSuite
 from rotest.common.constants import MAGENTA
-from rotest.core.test_filter import match_tags
+from rotest.core.filter import match_tags
```
Comment (UnDarkle) [inline src/rotest/core/utils/common.py]: reorder

File: src/rotest/management/base_resource.py [hunk 2]
```diff
@@ -237,7 +236,6 @@ def enable_debug(self):
         debug(self.initialize, ignore_exceptions=[KeyboardInterrupt, BdbQuit])
```
Comment (UnDarkle) [inline src/rotest/management/base_resource.py]: we can call this method,
it should be in abstract_test.py -> request resources I think

File: src/rotest/common/django_utils/settings_common.py [hunk 3]
```diff
@@ -1,47 +0,0 @@
-"""Common Django configuration constants.
```
Comment (UnDarkle) [inline src/rotest/common/django_utils/settings_common.py]: why did you delete all the settings? I think leaving one with the default behavior would be nicer for the average user who wants to avoid all the django crap

File: src/rotest/management/base_resource.py (hunk 2 above)
Comment (osherdp) [reply src/rotest/management/base_resource.py]: Are you talking about the `reset` method or the `initialize`?
I couldn't find any occurrence of `reset` except on `ut_models.py`, and I deleted it as well

Comment (UnDarkle) [reply src/rotest/management/base_resource.py]: Currently the resources aren't debugged when using -D
Because we never call the enable_debug method of the resources after locking them
